content/en/platform/corda/4.6/open-source/node-operations-upgrade-cordapps.md

```diff
@@ -79,7 +79,7 @@ In order to perform the upgrade, follow the following steps:
 * Replace the existing JAR with the new one
 * Make any database changes required to any custom vault tables for the upgraded CorDapp. The database update for a
 CorDapp upgrade follows the same steps as database setup for a new CorDapp. Corda Enterprise users can use the Database
-Management Tool as detailed [here](../../4.6/enterprise/node/operating/node-operations-cordapp-deployment.md/).
+Management Tool as detailed [here](../../../../../en/platform/corda/4.6/enterprise/node/operating/node-operations-cordapp-deployment.md).
```

Comment:
```suggestion
Management Tool as detailed in [Installing CorDapps on a node](../../../../../en/platform/corda/4.6/enterprise/node/operating/node-operations-cordapp-deployment.md).
```
Just avoiding the use of "here" so the user knows exactly what they are clicking on.